packages/api/src/lib/prepareParams.js

```diff
@@ -131,6 +132,7 @@ module.exports = async (operation, body, metadata) => {
         // as a metadata object and organize into parameters.
         // eslint-disable-next-line no-param-reassign
         metadata = body;
+        metadataIntersected = true;
```

Comment:
Open to suggestions on a better variable name than this. Basically I just need to know when `body` was copied to `metadata`, where what the user sent was a metadata object but since they didn't send a body, and `body` is the first argument to an operation we need to rewrite it a bit.

It's confusing I know and I'm not explaining it properly.